src/status_im/contexts/settings/wallet/saved_addresses/view.cljs

```diff
@@ -21,8 +22,9 @@
   []
   (let [inset-top           (safe-area/get-top)
         customization-color (rf/sub [:profile/customization-color])
-        saved-addresses     []
+        saved-addresses     (rf/sub [:wallet/saved-addresses])
```

Comment:
I take the plan is to actually use saved addresses eventually (in that case this is fine for the time being of course), but if you only are checking whether there's any saved addresses, you should subscribe only to `(seq saved-addresses)`, so that the view re-renders only when you flip from 0 to 1 or viceversa (maybe I am missing where you use it though, but couldn't find it)

Reply:
actually `(boolean (seq saved-addresses))` should be the subscription, `(seq saved-addresses)` is not good enough